rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -111,17 +111,17 @@ components:
             $ref: '#/components/schemas/ErrorMessage'
 
   parameters:
-    Keyword:
+    QueryString:
       name: q
       in: query
       schema:
         type: string
         minLength: 1
         maxLength: 100
         pattern: .*\S.*
-        example: "house"
+        example: "contents LIKE 'metadata contents'"
       description: >
-        The keyword that will be used in the query.
+        The query string is used to find products with matching properties.
```

Comment:
I think this should mention CQL.
```
The OGC CommonQL string that will be used in the query (see the "Annex B - BNF Definition of OGC CommonQL (Normative)" section at http://docs.opengeospatial.org/is/12-168r6/12-168r6.html). Supported attribute names and boolean terms should be defined by the implementing server.
```
Or, if it doesn't mention CQL, there should be a note about what the query string means, and the format should be updated:
```
Supported query strings should be defined by the implementing server.
```

Reply:
I think it should be the latter. I originally had it mentioning OGC, but after talking with Aaron, I think it's better to have that in the README of the Search service.

Reply:
I'm okay with taking out OGC, but I think it's still a good idea to mention CQL.